python_modules/libraries/dagster-dg/dagster_dg/context.py

```diff
@@ -898,3 +902,23 @@ def _get_dg_pypi_version_info(context: DgContext) -> Optional[DgPyPiVersionInfo]
                 context.config.cli.suppress_warnings,
             )
     return version_info
+
+
+def _validate_dagster_dg_and_dagster_version_compatibility(context: DgContext) -> None:
```

Comment:
included by accident?

Reply:
was part of a stack and just needed to be rebased